python/tank/platform/environment.py

```diff
@@ -36,14 +37,18 @@ class Environment(object):
     Don't construct this class by hand! Instead, use the
     pipelineConfiguration.get_environment() method.
 
+    This class contains immutable methods only, e.g. you can only read from
+    the yaml file. If you want to modify the yaml content, create a 
+    WritableEnvironment instance instead.
     """
 
     def __init__(self, env_path, pipeline_config, context=None):
         """
         Constructor
         """
-        self.__env_path = env_path
```

Comment:
Is there an official stance on the usage of _ vs __ in member names? At the module level the __ is probably not necessary, but I thought having _ and __ allowed us to state if something was meant to be overridable by a derive class or not.